### DISCUSSION TITLE
Add --external option to vercel dev

### DISCUSSION THREAD

**Discussion by mellunar** [vercel/vercel#8114] [Ideas]
I would like to suggest adding an `--external` option to `vercel dev` command. It would allow us to easily develop and check results on mobile devices by accessing the preview on local network IP instead of localhost, something like `ng serve --host` or `ionic serve --external`.

**Comment by amyegan** [comment]
Hi @mellunar. Thanks for sharing this idea!

Just in case this is currently causing friction for you or anyone else who comes across this, I want to share that there is a way to work around this for now. Using Angular as an example, adding a script like `"dev": "ng serve --host 0.0.0.0"` to your `packages.json` will make use of that host when you run `vercel dev` on that repo.
